packages/forms-web-app/src/lib/appeals-api-wrapper.js

```diff
@@ -97,6 +97,10 @@ exports.getExistingAppeal = async (appealId) => {
 	return handler(`/api/v1/appeals/${appealId}`);
 };
 
+exports.getAppealsByEmail = async (email) => {
+	return handler(`/api/v1/appeals/email/${email}`, 'GET');
+};
+
```

Comment:
We don't need this any more do we? Let's remove it and the backend code. It could be a very expensive query so we don't want to call it accidentally!